.github/workflows/test.yml

```diff
@@ -63,3 +63,32 @@ jobs:
 
       - name: Stop and remove container
         run: docker compose down
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry==1.8.1
+          poetry install --no-root
+
+      - name: Run isort
+        run: |
+          poetry run isort --check-only .
+
+      - name: Run black
+        run: |
+          poetry run black --check .
+
+      - name: Run pylint
+        run: |
+          poetry run pylint **/*.py
```

Comment:
Not sure if we want linting in workflow named `test.yml`. Maybe we can create new workflow for linting?